schemas/ispyb/stored_programs/sp_insert_dc_plan.sql

```diff
@@ -0,0 +1,100 @@
+DELIMITER ;;
+CREATE OR REPLACE DEFINER=`ispyb_root`@`%` PROCEDURE `insert_dc_plan`(
+	OUT p_id int(11) unsigned,
+	p_diffactionPlanId int(10),
+	p_name varchar(20),
+	p_experimentKind enum('Default','MXPressE','MXPressO','MXPressE_SAD','MXScore','MXPressM','MAD','SAD','Fixed','Ligand binding','Refinement','OSC','MAD - Inverse Beam','SAD - Inverse Beam','MESH','XFE','Stepped transmission','XChem High Symmetry','XChem Low Symmetry','Commissioning'),
+	p_observedResolution float,
+	p_minimalResolution float,
+	p_exposuretime float,
+	p_oscillationRange float,
+	p_maximalResolution float,
+	p_screeningResolution float,
+	p_radiationSensitivity float,
+	p_anomalousScatterer varchar(255),
+	p_preferredBeamSizeX float,
+	p_preferredBeamSizeY float,
+	p_preferredBeamDiameter float,
+	p_comments varchar(1024),
+	p_DIFFRACTIONPLANUUID varchar(1000),
```

Comment:
Although the corresponding column is in all-caps, I think the param name should follow the camelCase convention like the other params.